includes/Core/Admin_Bar/Admin_Bar.php

```diff
@@ -149,8 +149,21 @@ public function is_active() {
 			return false;
 		}
 
-		$post = get_post();
-		if ( ! $post ) {
+		// Shows admin bar on homepage when front page mode is latest blog posts.
+		if ( 'posts' === get_option( 'show_on_front' ) &&
+			is_home() &&
+			current_user_can( Permissions::VIEW_DASHBOARD ) ) {
+			return true;
+		}
```

Comment:
See above, this condition should be removed.

Reply:
Address in https://github.com/google/site-kit-wp/pull/73/commits/149af340e745c69f727713b5f99892abc94737d3